corsixth-appdata.patch

```diff
@@ -0,0 +1,55 @@
+From baec9533810026e55554ce7a6dc8e1817173ca88 Mon Sep 17 00:00:00 2001
+From: Toby <mrtobylane@gmail.com>
+Date: Wed, 26 Aug 2020 00:43:31 +0100
+Subject: [PATCH] Update appdata.xml (#1693)
+
+---
+ ...ppdata.xml => com.corsixth.corsixth.appdata.xml} | 13 +++++++------
+ 1 file changed, 7 insertions(+), 6 deletions(-)
+ rename CorsixTH/{com.corsixth.CorsixTH.appdata.xml => com.corsixth.corsixth.appdata.xml} (88%)
+
+diff --git a/CorsixTH/com.corsixth.CorsixTH.appdata.xml b/CorsixTH/com.corsixth.corsixth.appdata.xml
+similarity index 88%
+rename from CorsixTH/com.corsixth.CorsixTH.appdata.xml
+rename to CorsixTH/com.corsixth.corsixth.appdata.xml
+index 68819ccd94..b252d5bc78 100644
+--- a/CorsixTH/com.corsixth.CorsixTH.appdata.xml
++++ b/CorsixTH/com.corsixth.corsixth.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+-  <id>com.corsixth.CorsixTH</id>
++  <id>com.corsixth.corsixth</id>
```

Comment:
So why not using this ID instead to not even have to rename?

Reply:
Upstream changed their ID from `com.corsixth.CorsixTH` to `com.corsixth.corsixth`, but it happened after their latest release: https://github.com/CorsixTH/CorsixTH/issues/1416#issuecomment-835636263

I rename for this release, but it won't be necessary for future releases.

Reply:
ok. make sense.

Reply:
You should add the old id to `<provides><id>oldId</id></provides>`  so that the stores don't show the application twice

Reply:
Done, the patch has been submitted upstream here: https://github.com/CorsixTH/CorsixTH/pull/1871